### PR TITLE
add some documentation around making your own login pages

### PR DESCRIPTION
**Pull request by jasonblewis**
some suggestions for the documentation, partly to resolve https://github.com/PerlDancer/Dancer2-Plugin-Auth-Extensible/issues/59

### CODE REVIEW

File: lib/Dancer2/Plugin/Auth/Extensible.pm [hunk 1]
```diff
@@ -1290,10 +1290,44 @@ handle the route. Note that it must be a fully qualified sub. E.g.
 
 Then in your code you might simply use a template:
 
+    sub login_page_handler {
+        my $return_url = query_parameters->get('return_url');
+        template
+            'account/login',
+            { title => 'Sign in',
+              return_url => query_parameters->get('return_url'),
```
Comment (SysPete) [inline lib/Dancer2/Plugin/Auth/Extensible.pm]: I guess you meant this line to use the $return_url var?

Comment (jasonblewis) [inline lib/Dancer2/Plugin/Auth/Extensible.pm]: well spotted! sorry about that, I'll fix it up tomorrow and update the PR

**Comment by jasonblewis** [comment]
how does that look?

**Comment by SysPete** [comment]
Many thanks @jasonblewis! Apologies for the delay.

**Pull request merged**